src/backend/core/api/viewsets.py

```diff
@@ -399,3 +400,80 @@ def perform_update(self, serializer):
                 raise exceptions.ValidationError({"role": message})
 
         serializer.save()
+
+
+class InvitationViewset(
+    mixins.CreateModelMixin,
```

Comment:
We might need additional validation checks. ⚠️

When generating invitations, I shouldn't be able to:
- Invite someone using an email already associated with a team member.
- Invite someone using an email linked to an existing user.

Let's enhance our validation to ensure a smoother process!